f8a_worker/workers/graphaggregator.py

```diff
@@ -52,16 +52,20 @@ def execute(self, arguments):
                                      .format(manifest['filename']))
             out["details"][0]['manifest_file'] = manifest['filename']
             out["details"][0]['ecosystem'] = manifest['ecosystem']
-
+            out["details"][0]['manifest_file_path'] = manifest['filepath']
```

Comment:
please prefix the manifest_file_path with the manifest_name and set that new string as "manifest_name"